features/steps/thens/existance.py

```diff
@@ -69,4 +69,10 @@ def step_impl(context, inst):
 
     if not any(recursive_flatten(inst)):
         expected = get_previous_step_before_assertion(context)
-        yield ValidationOutcome(instance_id=inst, expected = expected, observed='Nonexistent', severity = OutcomeSeverity.ERROR)
\ No newline at end of file
+        yield ValidationOutcome(instance_id=inst, expected = expected, observed='Nonexistent', severity = OutcomeSeverity.ERROR)
+
+@gherkin_ifc.step("The IFC model contains information on {functional_part_description}")
+def step_impl(context, inst, functional_part_description):
+    # This rule always passes and is used solely to trigger activation of the the rule if an instance linked to the functional part is present
+    if False:
+        yield ValidationOutcome(inst=inst, severity=OutcomeSeverity.ERROR)
```

Comment:
```suggestion
    pass
```

I would just write it like this. (Otherwise for example you trigger code quality warnings about constant boolean expressions and unreachable code).

Reply:
That's what I implemented initially, but I ran into some errors in the decorator. We're iterating over yielded results, however when we only include 'pass' it seems that behave returns a None, causing an error (as yielding over None doesn't work). 
![image](https://github.com/user-attachments/assets/a4c90b00-a870-4698-aa26-989efbe532fa)